ccmlib/cluster.py

```diff
@@ -347,10 +347,10 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_
 
         return started
 
-    def stop(self, wait=True, gently=True, wait_other_notice=False):
+    def stop(self, wait=True, gently=True, wait_other_notice=False, wait_iterations=7):
```

Comment:
I think its better to have a time limit then iterations - its hard to convert one to the other

Reply:
The comment explains the conversion (2^wait_iterations-1 seconds). 

I wanted my change to change absolutely nothing in the existing behavior of the function (which did exactly 7 iterations in successively doubled sleeps up to 127 seconds) when this option is set to the default.

But there's something else I can do that will avoid any change to the default case: ask for the wait_seconds, not iterations, and always do 7 iterations, i.e., the sleep will start with 1.0*wait_seconds/127 instead of 0. Would you prefer this?